examples/react-vanilla-spa/server.js

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
+import { fileURLToPath } from 'node:url'
+import Fastify from 'fastify'
+import FastifyVite from '@fastify/vite'
+
+export async function main (dev) {
+  const server = Fastify()
+
+  await server.register(FastifyVite, {
+    root: import.meta.url,
```

Comment:
Do we want to start using `import.meta.dirname` over `.url` in the examples?

Reply:
Yes we do!